test/conversion_test.go

```diff
@@ -675,22 +675,32 @@ func TestTaskCRDConversion(t *testing.T) {
 	v1beta1TaskName := helpers.ObjectNameForTest(t)
 	v1beta1Task := parse.MustParseV1beta1Task(t, fmt.Sprintf(v1beta1TaskYaml, v1beta1TaskName, namespace))
 	v1TaskExpected := parse.MustParseV1Task(t, fmt.Sprintf(v1TaskYaml, v1beta1TaskName, namespace))
+	v1beta1TaskRoundTripExpected := parse.MustParseV1beta1Task(t, fmt.Sprintf(v1beta1TaskYaml, v1beta1TaskName, namespace))
```

Comment:
Thanks for pointing out!

Updated Task and Pipeline to compare with original CRD created, while TaskRun and PipelineRun would need new variable since they have expected status populated with different formatted naming using sprintf